src/event.js

```diff
@@ -1,4 +1,37 @@
-var oldLoad = jQuery.fn.load;
+var oldLoad = jQuery.fn.load,
+	originalFix = jQuery.event.fix;
+
+// Skip this when testing older 3.0 betas
```

Comment:
Hmm, that would be better but maybe `checking`? Maybe that's just me but when I see `testing` in a comment I think I'm in the test code, not source. :)